lib/ansible/modules/windows/win_domain_user.ps1

```diff
@@ -264,6 +306,11 @@ try {
             $user_groups += $group.name
         }
         $result.groups = $user_groups
+        Foreach ($attribute in $attributes.GetEnumerator()){
```

Comment:
We want to avoid this behaviour now, are you able to remove this section.

Reply:
Which behaviour?
remove the whole loop?

Reply:
Done